<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>gestion_employee</class>
 <widget class="QWidget" name="gestion_employee">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1050</width>
    <height>680</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1051</width>
     <height>680</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>2</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Ajouter un employee</string>
    </attribute>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>50</y>
       <width>471</width>
       <height>361</height>
      </rect>
     </property>
     <property name="title">
      <string>Ajout</string>
     </property>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>60</y>
        <width>56</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Nom</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>110</y>
        <width>56</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Prenom</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>160</y>
        <width>56</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Cin</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_nom">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>60</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background: #BBACAC;
color: #F5FFFA;
background-color: #BBACAC; border: none;
border-radius: 10px;
icon-size: 30px;
font: 14pt &quot;Bahnschrift&quot;;</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_prenom">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>110</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background: #BBACAC;
color: #F5FFFA;
background-color: #BBACAC; border: none;
border-radius: 10px;
icon-size: 30px;
font: 14pt &quot;Bahnschrift&quot;;</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_cin">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>160</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background: #BBACAC;
color: #F5FFFA;
background-color: #BBACAC; border: none;
border-radius: 10px;
icon-size: 30px;
font: 14pt &quot;Bahnschrift&quot;;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>210</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Identifiant</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>260</y>
        <width>56</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Tache</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_id">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>210</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background: #BBACAC;
color: #F5FFFA;
background-color: #BBACAC; border: none;
border-radius: 10px;
icon-size: 30px;
font: 14pt &quot;Bahnschrift&quot;;</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_tache">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>260</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background: #BBACAC;
color: #F5FFFA;
background-color: #BBACAC; border: none;
border-radius: 10px;
icon-size: 30px;
font: 14pt &quot;Bahnschrift&quot;;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pb_ajouter">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>140</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background: #26C4EC;
color: #F5FFFA;
background-color: #26C4EC; border: none;
border-radius: 10px;
icon-size: 30px;
font: 14pt &quot;Bahnschrift&quot;;</string>
      </property>
      <property name="text">
       <string>Ajouter</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_age">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>310</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background: #BBACAC;
color: #F5FFFA;
background-color: #BBACAC; border: none;
border-radius: 10px;
icon-size: 30px;
font: 14pt &quot;Bahnschrift&quot;;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>310</y>
        <width>56</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>age</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>-270</x>
        <y>-70</y>
        <width>1041</width>
        <height>680</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <zorder>label_10</zorder>
     <zorder>label</zorder>
     <zorder>label_2</zorder>
     <zorder>label_3</zorder>
     <zorder>le_nom</zorder>
     <zorder>le_prenom</zorder>
     <zorder>le_cin</zorder>
     <zorder>label_4</zorder>
     <zorder>label_5</zorder>
     <zorder>le_id</zorder>
     <zorder>le_tache</zorder>
     <zorder>pb_ajouter</zorder>
     <zorder>le_age</zorder>
     <zorder>label_6</zorder>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>1051</width>
       <height>680</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>../../Bureau/Planet-Zoo-Keeper-Post-Header-If-Featured-747x309.jpg</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <zorder>label_12</zorder>
    <zorder>groupBox</zorder>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Afficher les employee</string>
    </attribute>
    <widget class="QTableView" name="tab_employee">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>300</y>
       <width>591</width>
       <height>281</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background: #BBACAC;
color: #F5FFFA;
background-color: #BBACAC; border: none;
border-radius: 10px;
icon-size: 30px;
font: 14pt &quot;Bahnschrift&quot;;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_14">
     <property name="geometry">
      <rect>
       <x>650</x>
       <y>130</y>
       <width>101</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background: #26C4EC;
color: #F5FFFA;
background-color: #26C4EC; border: none;
border-radius: 10px;
icon-size: 30px;
font: 14pt &quot;Bahnschrift&quot;;</string>
     </property>
     <property name="text">
      <string>   Nom</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_15">
     <property name="geometry">
      <rect>
       <x>650</x>
       <y>170</y>
       <width>121</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background: #26C4EC;
color: #F5FFFA;
background-color: #26C4EC; border: none;
border-radius: 10px;
icon-size: 30px;
font: 14pt &quot;Bahnschrift&quot;;</string>
     </property>
     <property name="text">
      <string>   Prenom</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_16">
     <property name="geometry">
      <rect>
       <x>650</x>
       <y>220</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background: #26C4EC;
color: #F5FFFA;
background-color: #26C4EC; border: none;
border-radius: 10px;
icon-size: 30px;
font: 14pt &quot;Bahnschrift&quot;;</string>
     </property>
     <property name="text">
      <string>    Cin</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_17">
     <property name="geometry">
      <rect>
       <x>650</x>
       <y>299</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background: #26C4EC;
color: #F5FFFA;
background-color: #26C4EC; border: none;
border-radius: 10px;
icon-size: 30px;
font: 14pt &quot;Bahnschrift&quot;;</string>
     </property>
     <property name="text">
      <string> Identifiant</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_18">
     <property name="geometry">
      <rect>
       <x>650</x>
       <y>260</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background: #26C4EC;
color: #F5FFFA;
background-color: #26C4EC; border: none;
border-radius: 10px;
icon-size: 30px;
font: 14pt &quot;Bahnschrift&quot;;</string>
     </property>
     <property name="text">
      <string> Tache</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="nv_nom">
     <property name="geometry">
      <rect>
       <x>760</x>
       <y>130</y>
       <width>113</width>
       <height>22</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background: #BBACAC;
color: #F5FFFA;
background-color: #BBACAC; border: none;
border-radius: 10px;
icon-size: 30px;
font: 14pt &quot;Bahnschrift&quot;;</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="nv_prenom">
     <property name="geometry">
      <rect>
       <x>760</x>
       <y>170</y>
       <width>113</width>
       <height>22</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background: #BBACAC;
color: #F5FFFA;
background-color: #BBACAC; border: none;
border-radius: 10px;
icon-size: 30px;
font: 14pt &quot;Bahnschrift&quot;;</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="nv_tache">
     <property name="geometry">
      <rect>
       <x>760</x>
       <y>260</y>
       <width>113</width>
       <height>22</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background: #BBACAC;
color: #F5FFFA;
background-color: #BBACAC; border: none;
border-radius: 10px;
icon-size: 30px;
font: 14pt &quot;Bahnschrift&quot;;</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="nv_cin">
     <property name="geometry">
      <rect>
       <x>760</x>
       <y>220</y>
       <width>113</width>
       <height>22</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background: #BBACAC;
color: #F5FFFA;
background-color: #BBACAC; border: none;
border-radius: 10px;
icon-size: 30px;
font: 14pt &quot;Bahnschrift&quot;;</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="nv_id">
     <property name="geometry">
      <rect>
       <x>760</x>
       <y>310</y>
       <width>113</width>
       <height>22</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background: #BBACAC;
color: #F5FFFA;
background-color: #BBACAC; border: none;
border-radius: 10px;
icon-size: 30px;
font: 14pt &quot;Bahnschrift&quot;;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pb_modifier">
     <property name="geometry">
      <rect>
       <x>930</x>
       <y>210</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background: #26C4EC;
color: #F5FFFA;
background-color: #26C4EC; border: none;
border-radius: 10px;
icon-size: 30px;
font: 14pt &quot;Bahnschrift&quot;;</string>
     </property>
     <property name="text">
      <string>Modifier</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pb_supprimer">
     <property name="geometry">
      <rect>
       <x>930</x>
       <y>280</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background: #26C4EC;
color: #F5FFFA;
background-color: #26C4EC; border: none;
border-radius: 10px;
icon-size: 30px;
font: 14pt &quot;Bahnschrift&quot;;</string>
     </property>
     <property name="text">
      <string>supprimer</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="cin_rech">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>261</y>
       <width>113</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background: #BBACAC;
color: #F5FFFA;
background-color: #BBACAC; border: none;
border-radius: 10px;
icon-size: 30px;
font: 14pt &quot;Bahnschrift&quot;;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pb_rech">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>260</y>
       <width>201</width>
       <height>28</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background: #26C4EC;
color: #F5FFFA;
background-color: #26C4EC; border: none;
border-radius: 10px;
icon-size: 30px;
font: 14pt &quot;Bahnschrift&quot;;</string>
     </property>
     <property name="text">
      <string>rechercher par cin</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pb_tri">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>220</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background: #26C4EC;
color: #F5FFFA;
background-color: #26C4EC; border: none;
border-radius: 10px;
icon-size: 30px;
font: 14pt &quot;Bahnschrift&quot;;</string>
     </property>
     <property name="text">
      <string>trier</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pb_stat">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>220</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background: #26C4EC;
color: #F5FFFA;
background-color: #26C4EC; border: none;
border-radius: 10px;
icon-size: 30px;
font: 14pt &quot;Bahnschrift&quot;;</string>
     </property>
     <property name="text">
      <string>statistique</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>650</x>
       <y>360</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background: #26C4EC;
color: #F5FFFA;
background-color: #26C4EC; border: none;
border-radius: 10px;
icon-size: 30px;
font: 14pt &quot;Bahnschrift&quot;;</string>
     </property>
     <property name="text">
      <string>age</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="nv_age">
     <property name="geometry">
      <rect>
       <x>760</x>
       <y>360</y>
       <width>113</width>
       <height>22</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background: #BBACAC;
color: #F5FFFA;
background-color: #BBACAC; border: none;
border-radius: 10px;
icon-size: 30px;
font: 14pt &quot;Bahnschrift&quot;;</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>260</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background: #26C4EC;
color: #F5FFFA;
background-color: #26C4EC; border: none;
border-radius: 10px;
icon-size: 30px;
font: 14pt &quot;Bahnschrift&quot;;</string>
     </property>
     <item>
      <property name="text">
       <string>par nom</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>par prenom</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>par age</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>1061</width>
       <height>751</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>../../Bureau/Planet-Zoo-Keeper-Post-Header-If-Featured-747x309.jpg</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <zorder>label_11</zorder>
    <zorder>tab_employee</zorder>
    <zorder>label_14</zorder>
    <zorder>label_15</zorder>
    <zorder>label_16</zorder>
    <zorder>label_17</zorder>
    <zorder>label_18</zorder>
    <zorder>nv_nom</zorder>
    <zorder>nv_prenom</zorder>
    <zorder>nv_tache</zorder>
    <zorder>nv_cin</zorder>
    <zorder>nv_id</zorder>
    <zorder>pb_modifier</zorder>
    <zorder>pb_supprimer</zorder>
    <zorder>cin_rech</zorder>
    <zorder>pb_rech</zorder>
    <zorder>pb_tri</zorder>
    <zorder>pb_stat</zorder>
    <zorder>label_7</zorder>
    <zorder>nv_age</zorder>
    <zorder>comboBox</zorder>
   </widget>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>chat </string>
    </attribute>
    <widget class="QPushButton" name="pb_server">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>220</y>
       <width>181</width>
       <height>28</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background: #26C4EC;
color: #F5FFFA;
background-color: #26C4EC; border: none;
border-radius: 10px;
icon-size: 30px;
font: 14pt &quot;Bahnschrift&quot;;</string>
     </property>
     <property name="text">
      <string>lancer le serveur</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>330</y>
       <width>51</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Message</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pb_envoyer">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>530</y>
       <width>71</width>
       <height>23</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background: #26C4EC;
color: #F5FFFA;
background-color: #26C4EC; border: none;
border-radius: 10px;
icon-size: 30px;
font: 14pt &quot;Bahnschrift&quot;;</string>
     </property>
     <property name="text">
      <string>Envoyer</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pb_connecter">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>530</y>
       <width>91</width>
       <height>23</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background: #26C4EC;
color: #F5FFFA;
background-color: #26C4EC; border: none;
border-radius: 10px;
icon-size: 30px;
font: 14pt &quot;Bahnschrift&quot;;</string>
     </property>
     <property name="text">
      <string>Connecter</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>258</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background: #BBACAC;
color: #F5FFFA;
background-color: #BBACAC; border: none;
border-radius: 10px;
icon-size: 30px;
font: 14pt &quot;Bahnschrift&quot;;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_nickname">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>258</y>
       <width>391</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background: #BBACAC;
color: #F5FFFA;
background-color: #BBACAC; border: none;
border-radius: 10px;
icon-size: 30px;
font: 14pt &quot;Bahnschrift&quot;;</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_message">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>530</y>
       <width>291</width>
       <height>19</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background: #BBACAC;
color: #F5FFFA;
background-color: #BBACAC; border: none;
border-radius: 10px;
icon-size: 30px;
font: 14pt &quot;Bahnschrift&quot;;</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="textEdit">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>300</y>
       <width>471</width>
       <height>222</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background: #BBACAC;
color: #F5FFFA;
background-color: #BBACAC; border: none;
border-radius: 10px;
icon-size: 30px;
font: 14pt &quot;Bahnschrift&quot;;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>1030</width>
       <height>680</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>../../Bureau/Planet-Zoo-Keeper-Post-Header-If-Featured-747x309.jpg</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <zorder>label_13</zorder>
    <zorder>pb_server</zorder>
    <zorder>label_8</zorder>
    <zorder>pb_envoyer</zorder>
    <zorder>pb_connecter</zorder>
    <zorder>label_9</zorder>
    <zorder>le_nickname</zorder>
    <zorder>le_message</zorder>
    <zorder>textEdit</zorder>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
